### ISSUE TITLE
website: Missing Info on Main Page?

### ISSUE THREAD

**Issue by markus2330** [ElektraInitiative/libelektra#1182]
Some more infos that likely should be on the main page:

- [x] fast and nodeps
- [x] 100% free software (BSD license)
- [x] cross platform

- [x] The text in "Sharing of Configuration Snippets" and "Graphical User Interface" is a bit hard to read (too long lines)

- [x] I would also suggest a "Share snippet" next to "Convert snippet" that leads to the login page?

- [x]  gray lines between the different UIs?

- [x] Other applications has some space where we could put logos (oyranos+Distributions that have Elektra packages)

- [X] I like the large icons in the upper 6 items more than the small icons below the asciinema (nothing to do).

**Comment by Namoshek** [comment]
Have you used another resolution for the new asciinema? It looks weird in my opinion.

**Comment by markus2330** [comment]
> Have you used another resolution for the new asciinema? It looks weird in my opinion.

It is 80x25, what should it be? (There are some issues with sudo anyway, so recording is not an issue.)

> Putting larger icons on the other points too makes the page look crowded, it doesn't look very good.

Ok.

> Btw. I don't get why you think the sentences of "Graphical User Interface" are long?

For example, the first line of the paragraph is: "This website does not only contain the news and documentation around Elektra, but does also comprise a" which is 17 words or 104 characters.

For example, http://baymard.com/blog/line-length-readability recommends less than 75 characters.

It might be okay if not all the other paragraphs had so short lines.

**Comment by markus2330** [comment]
I added one more point above. Maybe its possible to have some logos of oyranos and distros that have Elektra packages?

**Comment by Namoshek** [comment]
> It is 80x25, what should it be? (There are some issues with sudo anyway, so recording is not an issue.)

The previous one was perfect from the size (I don't know what size it was). The new one has 25 visible lines, the old one 23 (quickly counted them). Maybe this info is enough for you to adjust?

> For example, the first line of the paragraph is: "This website does not only contain the news and documentation around Elektra, but does also comprise a" which is 17 words or 104 characters.

Yes that is the Snippet Sharing, I changed the sentence (actually cut it into two sentences). But the Graphical User Interface section only has very short sentences, so I was confused. See:
 
> Elektra offers a GUI based on Qt Quick. It simplifies common tasks and offers easy access to the key database itself. A description for the GUI tool can be found here.

> I added one more point above. Maybe its possible to have some logos of oyranos and distros that have Elektra packages?

There is enough space to do so, but we have to see if it looks crowded. I think we are currently very near to a crowded main page. ^^

**Comment by markus2330** [comment]
> Maybe this info is enough for you to adjust?

Yes, this is enough info. Is there anything else to be changed in the frontend except of changing the link? (when we have it with 100x23?)

> Yes that is the Snippet Sharing, I changed the sentence (actually cut it into two sentences). 

I did not meant the sentence length (even though this was an issue, too), but the line length (when there is a line wrap/new line).

> we are currently very near to a crowded main page. ^^

I am totally for having more white space ;) Or some fading in/out when scrolling?

I added the points to #1151 and closed this issue.
